### PR TITLE
DB schema and Picture Function

### PR DESCRIPTION
**Pull request by Nasser-Eddin-Nasser**

### CODE REVIEW

**Review by dalex135 — APPROVED** [review]
Checked and it's working perfectly.

**Review by vishwasanavatti — APPROVED** [review]
Tested locally, It works fine. 
Also I made a merge commit by mistake. I guess I have reverted the changes but please do cross checks. Sorry for the inconvenience.

**Review by vishwasanavatti — APPROVED** [review]
Tested again after the change. Code is clean, really liked it. I could not test the error function as I have local issue for running image. Anyways it can be merged to master.

File: Backend/src/main/java/Digitact/Backend/Model/Image/AppImage.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    @OneToMany(mappedBy = "appImage", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @Fetch(FetchMode.JOIN)
+    private List<Block> blocks;
+
+    @Column(name = "type")
+    private ImageType type;
+
+    @JsonIgnore
+    public AppImage(List<Block> blocks) {
+        this.id = this.id = UUID.randomUUID().toString();
```
Comment (Nasser-Eddin-Nasser) [inline Backend/src/main/java/Digitact/Backend/Model/Image/AppImage.java]: fixe this

**Comment by vishwasanavatti** [comment]
I tested for error function and it works.

**Pull request merged**